types.go

```diff
@@ -69,10 +82,8 @@ type SectorInfo struct {
 
 	// Debug
 	LastErr string
-}
 
-func (t *SectorInfo) upd() *sectorUpdate {
-	return &sectorUpdate{id: t.SectorID, nonce: t.Nonce}
+	Log []Log
```

Comment:
What rolls downstairs, alone or in pairs...

Reply:
![index](https://user-images.githubusercontent.com/884507/73298984-2f869a00-41c3-11ea-8b89-c1c9cecd9403.jpg)